templates/default/jaas-activedirectory.conf.erb

```diff
@@ -29,7 +29,11 @@ activedirectory {
 <% end -%>
     reportStatistics="<%=@ldap[:reportstatistics]%>";
 
-    org.eclipse.jetty.plus.jaas.spi.PropertyFileLoginModule required
+    <% if Gem::Requirement.new('< 2.7.0').satisfied_by?(Gem::Version.new(@rundeck_version)) %>
```

Comment:
@rundeck_version is available in the template. You are seeing the wrong file. check this out https://github.com/nilroy/rundeck/blob/2018dfc7219cd187109fca9f3e21dfd1ae4c4413/recipes/server_install.rb#L164